src/main/io/displayport_msp_osd.c

```diff
@@ -439,6 +467,11 @@ displayPort_t* mspOsdDisplayPortInit(const videoSystem_e videoSystem)
             screenRows = DJI_ROWS;
             screenCols = DJI_COLS;
             break;
+        case VIDEO_SYSTEM_AVATAR:
+            currentOsdMode = HD_5018;
```

Comment:
It probably won't affect the code, but every time a new resolution was added, we had a new HD_ XXXX definition. Since avatr seesmt o be 53x20, we probably need to add HD_5320

It seems to only matter for (osdVideoSystem == VIDEO_SYSTEM_HDZERO)

Reply:
Yep, only for HDZero, so they can switch between 50 x 18 and 30 x 16.